ui/job-view/pushes/Push.jsx

```diff
@@ -337,6 +339,45 @@ class Push extends React.Component {
     );
   };
 
+  showFuzzyJobs = async () => {
+    const { push, repoName, getGeckoDecisionTaskId, notify } = this.props;
+    const excludedJobNames = /(balrog|beetmover|bouncer-locations-firefox|build-docker-image|build-(.+)-nightly|build-(.+)-upload-symbols|checksums|cron-bouncer|dmd|fetch|google-play-strings|push-to-release|mar-signing|nightly|packages|release-bouncer|release-early|release-final|release-secondary|release-snap|release-source|release-update|repackage-l10n|repo-update|searchfox|sign-and-push|test-(.+)-devedition|test-linux(32|64)(-asan|-pgo|-qr)?\/(opt|debug)-jittest|test-macosx64-ccov|test-verify|test-windows10-64-ux|toolchain|upload-generated-sources)/;
```

Comment:
@edmorley @camd Not super happy with that super-long regex for trimming the list down, but I don't know enough regex-fu to make it any shorter. Any chance either of you know if I could trim that down any, or maybe break it onto multiple lines so it's a bit easier to read? 

I basically just want any job name that contains any of those strings to be removed.

Reply:
[Tagged Templates](https://stackoverflow.com/questions/12317049/how-to-split-a-long-regular-expression-into-multiple-lines-in-javascript#answers) let me break this into multiple lines.